std/complex.d

```diff
@@ -525,26 +525,26 @@ if (isFloatingPoint!T)
     assert(rec2a.im == 0.0);
 
     auto rec1b = (-1.0) ^^ c1;
-    assert(approxEqual(abs(rec1b), std.math.exp(-PI * c1.im), EPS));
+    assert(isClose(abs(rec1b), std.math.exp(-PI * c1.im), EPS));
     auto arg1b = arg(rec1b);
     /* The argument _should_ be PI, but floating-point rounding error
      * means that in fact the imaginary part is very slightly negative.
      */
-    assert(approxEqual(arg1b, PI, EPS) || approxEqual(arg1b, -PI, EPS));
+    assert(isClose(arg1b, PI, EPS) || isClose(arg1b, -PI, EPS));
 
     auto rec2b = (-1.0) ^^ c2;
-    assert(approxEqual(abs(rec2b), std.math.exp(-2 * PI), EPS));
-    assert(approxEqual(arg(rec2b), PI_2, EPS));
+    assert(isClose(abs(rec2b), std.math.exp(-2 * PI), EPS));
+    assert(isClose(arg(rec2b), PI_2, EPS));
 
     auto rec3a = 0.79 ^^ complex(6.8, 5.7);
     auto rec3b = complex(0.79, 0.0) ^^ complex(6.8, 5.7);
-    assert(approxEqual(rec3a.re, rec3b.re, EPS));
-    assert(approxEqual(rec3a.im, rec3b.im, EPS));
+    assert(isClose(rec3a.re, rec3b.re, 1e-14));
+    assert(isClose(rec3a.im, rec3b.im, 1e-14));
```

Comment:
Here, and in several more places, the absolute difference check with accuracy of 1e-2 was used, instead of the intended relative difference check with accuracy of `EPS`. With `isClose` this test is disabled until you explicitly ask for. The given accuracy of the relative difference check was too much for 32bit computers, so I reduced it a little bit. Might still be too much on computers where `real==double`.